### PR TITLE
Hamburger  mobile menu

### PR DESCRIPTION
**Pull request by shobeee**
it open menu when we tap menu button
it closes menu when we tap close button
it directs us to the page that we select in the menu

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @shobeee  

## While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 🔴 🔴 

You have submitted a project, that doesn't pass all the linters

Please fix the stylelint errors on your code you could run the following code in your command line

`npx stylelint "**/*.{css,scss}" --fix` "NOTE THE LAST --FIX WORD, this fix minor errors of the format in your code"

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Omar-Muhamad — COMMENTED** [review]
# Invalid ❌

Hi Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Your Code Review Request is marked as invalid for these reasons:
As the previous reviewer mentioned, You have submitted a project, that doesn't pass all the linters

Please fix the stylelint errors on your code you could run the following code in your command line

`npx stylelint "**/*.{css,scss}"` --fix "NOTE THE LAST --FIX WORD, this fix minor errors of the format in your code"

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -6,23 +6,19 @@
...
+            <li class="nav-links"><a id="about-header" href="#about-me">About</a></li>
+            <li class="nav-links"><a id="contact-header" href="#form">Contact</a></li>
+            <li id="letter-icon"><a href="#"><img src="images/letter-icon.svg" id="letter-icon"></a></li>
         </ul>
-            <div class="hamburger" id="hamburger">
-            <span class="bar"></span>
-            <span class="bar"></span>
-            <span class="bar"></span>
-        </div>
+        <span class="material-symbols-outlined"style="font-size: 36px;" id="menu-1">menu</span>
```
Comment (Abenezer-Tilahun) [inline index.html:21]: - [ ] Good Job for Implementing all the functionality But You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly change "Menu" and "Close" button with `hamburger` and `close` button Just Like the Figma design.

Your Project Mobile Menu Look likes this 
Capture

Capture

The Figma design Look like this : 

Capture

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @shobeee , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional  README 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Kindly Update your README file with new [README template](https://github.com/microverseinc/readme-template)

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @shobeee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: Good PR summary.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**